x-pack/filebeat/module/microsoft/m365_defender/config/defender.yml

```diff
@@ -50,9 +50,6 @@ tags: {{ .tags | tojson }}
 publisher_pipeline.disable_host: {{ inList .tags "forwarded" }}
 
 processors:
-  - decode_json_fields:
```

Comment:
Wouldn't this change break some users set up if they are using LS?

Reply:
That is correct, I had some issues with CI, which I guess turned out to be related to something else, and I guess since this is an existing module I will convert it, since we are removing the processors only on the package side.